ui/src/containers/ClusterMonitoring.js

```diff
@@ -238,7 +255,16 @@ const ClusterMonitoring = props => {
           </Tooltip>
         </RightClusterStatusContainer>
       </ClusterStatusTitleContainer>
-
+      {clusterStatus.value === CLUSTER_STATUS_UNKNOWN &&
+      cluster.prometheusPodStatusMessage ===
```

Comment:
Indeed, I'm not sure if this value will always match this string. Let's be slightly more flexible:

```javascript
cluster.prometheusPodStatusMessage.includes(`didn't find available persistent volumes to bind`)
```